### PR TITLE
update cnab.rs to CNAB Core 1.0

### PR DESCRIPTION
**Pull request by technosophos**
This PR contains a number of small changes that bring this library up to date with the spec.

Signed-off-by: Matt Butcher <matt.butcher@microsoft.com>

### CODE REVIEW

File: src/tests.rs [hunk 1]
```diff
@@ -405,7 +386,7 @@ fn test_bundle_parse_error() {
 // Test loading a bundle from a file
 #[test]
 fn test_bundle_deserialize() {
-    let bun = Bundle::from_file("testdata/bundle.json").unwrap();
+    let bun = Bundle::from_file("testdata/bundle.json").expect("parse testdata/bundle.json");
 
     assert_that(&bun.name).is_equal_to("helloworld".to_string());
     assert_that(&bun.schema_version).is_equal_to("v1.0.0-WD".to_string());
```
Comment (radu-matei) [inline src/tests.rs]: Should we also update the test bundle to v1.0.0?

File: src/cnab.rs [hunk 2]
```diff
@@ -265,3 +278,19 @@ pub struct Destination {
...
 }
+
+/// A value that is produced by running an invocation image
+///
+/// Complies to CNAB Core 1.0
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Output {
+    /// An optional exhaustive list of actions producing this output
+    pub apply_to: Option<String>,
```
Comment (radu-matei) [inline src/cnab.rs]: This should be `Option<Vec<String>>` - see https://github.com/deislabs/cnab-spec/blob/master/101-bundle-json.md#outputs

File: src/tests.rs (hunk 1 above)
Comment (technosophos) [reply src/tests.rs]: Good catch! Yes we should.

File: src/cnab.rs (hunk 2 above)
Comment (technosophos) [reply src/cnab.rs]: You are correct. Thanks

**Comment by technosophos** [comment]
Superseded by #23

**Pull request closed without merging**